src/csharp/Grpc.Core.Tests/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "buildOptions": {
+    "compile": "**/*.cs",
+    "copyToOutput": {
+      "mappings": {
+        "nativelibs/windows_x64/grpc_csharp_ext.dll": "../../../vsprojects/x64/Debug/grpc_csharp_ext.dll",
+        "nativelibs/windows_x86/grpc_csharp_ext.dll": "../../../vsprojects/Debug/grpc_csharp_ext.dll"
+      }
+    },
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Grpc.Core": {
+      "version": "0.0.1",
+      "taget": "project"
+    },
+    "Newtonsoft.Json": "8.0.3",
+    "NUnit": "3.2.0",
+    "NUnitLite": "3.2.0-*"
+  },
+  "frameworks": {
+    "net45": { },
+    "netstandard1.5": {
+      "imports": [
+        "portable-net45"
+      ],
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-rc2-24027"
+      }
+    }
+  },
+  "runtimes": {
```

Comment:
Add more runtimes so you can test on other platforms? Or does it all break horribly at the moment?